scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -398,10 +398,11 @@ class FossId internal constructor(
         url: String,
         revision: String? = null
     ): List<Scan> = filter {
+        val isArchived = it.isArchived ?: false
```

Comment:
Does it make sense to instead modify the `IntBooleanDeserializer` to deserialize to `false` in case of a `null` value? Is that the right thing to do for all users of `IntBooleanDeserializer`?